copy.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+// #include "uplink_definitions.h"
+import "C"
+
+//export uplink_copy_object
+// uplink_copy_object copies object to a same/different bucket and key.
+func uplink_copy_object(project *C.UplinkProject, old_bucket_name, old_object_key, new_bucket_name, new_object_key *C.uplink_const_char,
+	options *C.UplinkMoveObjectOptions) *C.UplinkError { //nolint:golint
```

Comment:
UplinkMoveObjectOptions -> UplinkCopyObjectOptions

Please also add the type to uplink_definitions.h